pallets/loans/Cargo.toml

```diff
@@ -34,18 +34,18 @@ pallet-balances = { git = "https://github.com/paritytech/substrate",  default-fe
 pallet-uniques = { git = "https://github.com/paritytech/substrate",  default-features = false , optional = true, branch = "polkadot-v0.9.20" }
 pallet-pools = { path = "../pools", optional = true, default-features = false}
 
-orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", default-features = false, optional = true, branch = "master" }
+orml-tokens = { git = "https://github.com/NunoAlexandre/open-runtime-module-library", default-features = false, optional = true, rev = "3c81bb8d0d6bcf58606e11cdf17f20f9abc8fb13" }
```

Comment:
Using my own fork with the new asset-registry features back-ported to polkadot v0.9.20